### DISCUSSION TITLE
Use taiga-ui with SASS

### DISCUSSION THREAD

**Discussion by fynnfeldpausch** [taiga-family/taiga-ui#92] [Q&A] [answered]
Has anyone tried using taiga-ui in a SASS project? Is importing the CSS build the only option here or are there other possibilities. Would love to hear some feedback.

**Comment by waterplea** [comment, marked as answer]
Angular should be able to handle it for you. Just add our global styles and theme in `angular.json`:
![image](https://user-images.githubusercontent.com/11832552/104365178-bc3b7c80-5528-11eb-8d16-c481ec250024.png)

**Reply by fynnfeldpausch** [reply]
Yeah, I know. Was just interested in the interaction between the two preprocessors when it comes to LESS variables, mixins, ...

**Reply by waterplea** [reply]
Yes, you will not be able to use our mixins in other preprocessors. As for variables, for colors we rely on native CSS custom properties, so there will be no issue there. I expect in future we might completely get rid of LESS variables in favor of CSS for easier customization.

**Reply by JODIN1727** [reply]
We are now  working on this implementation smoothly.. thank

**Reply by JODIN1727** [reply]
Thanks
